src/components/ConsortiumManagerNavigationPaneToggle/ConsortiumManagerNavigationPaneToggle.js

```diff
@@ -0,0 +1,20 @@
+import { PaneHeaderIconButton } from '@folio/stripes/components';
+
+import { useCallback } from 'react';
```

Comment:
Please re-order imports so the are in this order:
1. Third party libraries
2. Stripes imports
3. Local module imports

Please also check other files in this PR

Reply:
Good catch, thanks! Updated